R/CohortCharacterizationDiagnostics.R

```diff
@@ -45,7 +45,7 @@ getCohortCharacteristics <- function(connectionDetails = NULL,
                                      cohortDatabaseSchema = cdmDatabaseSchema,
                                      cohortTable = "cohort",
                                      cohortId,
-                                     covariateSettings = FeatureExtraction::createDefaultCovariateSettings()) {
+                                     covariateSettings) {
```

Comment:
Just curious: why no longer provide a default?

Reply:
When i added temporalCharacterization, i made this function `getCohortCharacteristics` support both temporal and non temporal characterization. 

I might be wrong here, but i think, i did not want to default it to non temporal covariate settings.

thinking back - i could have left it as is